com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -1212,6 +1226,21 @@ def set_parallel(self, width):
         self.oport.operator.config['width'] = streamsx.topology.graph._as_spl_json(width, int)
         return self
 
+    def set_consistent(self, consistent_config):
+        """ Indicates that the stream is the start of a consistent region.
+
+        Args:
+            consistent_config(consistent.ConsistentRegionConfig): the configuration of the consistent region.
+
+        Returns:
+            Stream: Returns this stream.
+        """
+
+        # add job control plane if needed
```

Comment:
Needs to be placable, otherwise a union followed y set_consistent will fail.

Reply:
I added _makePlaceable()